radixdlt/src/main/java/com/radixdlt/consensus/EpochManager.java

```diff
@@ -183,6 +185,33 @@ public void processEpochChange(EpochChange epochChange) {
 		queuedEvents.remove(nextEpoch);
 	}
 
+	private void logEpochChange(EpochChange epochChange, String message) {
+		if (log.isInfoEnabled()) {
```

Comment:
```java
if (!log.isInfoEnabled()) {
     return;
}
```

And then remove one level of indentation? :)

Reply:
Also not part of PR.